### ISSUE TITLE
iPhone Setup

### ISSUE THREAD

**Issue by mdb1** [mdb1/mdb1.github.io#55]
- Apps
- Do Not Disturb Focus Mode Automatic Schedule (with allowed apps + time sensitive notifications)
- Shortcuts Automation to set Color Filters on (black and white)
- Set Accessibility Shortcut to Color Filters (black and white)
- Opal vs ClearSpace (compare both and choose the best one)
- Notifications off for most apps
- No Social Media apps
